pkg/systemd/services/services.jsx

```diff
@@ -705,7 +705,7 @@ class ServicesPage extends React.Component {
                         return false;
 
                     if (currentTextFilter && unit.Description && unit.Description.toLowerCase().indexOf(currentTextFilter) == -1 &&
-                        unit.Id.indexOf(currentTextFilter) == -1)
+                        unit.Id.indexOf(currentTextFilter) == -1 && !(unit.Names || []).some(name => name.indexOf(currentTypeFilter) != -1))
```

Comment:
so this commit introduced this and then next commit dropped it?

Reply:
Mistake - I did want that it.